Gem/Code/Source/UserSettings/MultiplayerSampleUserSettings.cpp

```diff
@@ -180,6 +188,99 @@ namespace MultiplayerSample
         }
     }
 
+    bool MultiplayerSampleUserSettings::GetFullscreen()
+    {
+        bool fullscreen = false;
+
+        if (auto* registry = AZ::SettingsRegistry::Get(); registry != nullptr)
+        {
+            registry->Get(fullscreen, m_fullscreenKey.c_str());
+        }
+
+        return fullscreen;
+    }
+
+    void MultiplayerSampleUserSettings::SetFullscreen(bool fullscreen)
+    {
+        if (auto* registry = AZ::SettingsRegistry::Get(); registry != nullptr)
+        {
+            if (AZ::IConsole* console = AZ::Interface<AZ::IConsole>::Get(); console)
+            {
+                // Change the fullscreen state if we haven't created the window yet.
+                AZ::CVarFixedString commandString = AZ::CVarFixedString::format("r_fullscreen %u", fullscreen ? 1 : 0);
+                console->PerformCommand(commandString.c_str());
+
+                // Change the fullscreen state if the window already exists
+                AzFramework::NativeWindowHandle windowHandle = nullptr;
+                AzFramework::WindowSystemRequestBus::BroadcastResult(
+                    windowHandle,
+                    &AzFramework::WindowSystemRequestBus::Events::GetDefaultWindowHandle);
+
+                AzFramework::WindowRequestBus::Event(
+                    windowHandle,
+                    &AzFramework::WindowRequestBus::Events::SetFullScreenState, fullscreen);
+            }
+
+            registry->Set(m_fullscreenKey.c_str(), fullscreen);
+        }
+    }
+
+    AZStd::pair<uint32_t, uint32_t> MultiplayerSampleUserSettings::GetResolution()
+    {
+        uint64_t width = 1920;
```

Comment:
nit: Feel like these should be easier to find as defaults ie named defaults with reason for how set. Otherwise they are hidden inside code.

Reply:
I'm totally open to suggestions here. 

I could try and grab each value from its associated cvar, setreg setting, etc, though some things like master volume don't even have an exposed setting anywhere, and I think this class initializes too early to guarantee that it _could_ grab defaults from all of the associated places.

I could create a little const struct that contains all the defaults in it and place it at the top of the file.

I could try and read the defaults in from a data file or setreg file, but then that brings me back to the position of needing default values if those don't exist...